community/phpldapadmin/APKBUILD

```diff
@@ -1,17 +1,12 @@
 # Maintainer: Leonardo Arena <rnalrd@alpinelinux.org>
-_php=php5
```

Comment:
I'd prefer to do not remove it. Other packages use the same approach. It's handy for the next php version.

Reply:
I will revert template but package is broken cos at least php5-session is needed, olso I'd added opcache or apc in phpt

Reply:
Also I'm not sure zlib is bundled in php5 build like in php7

Reply:
Pushed updates